Jenkinsfile

```diff
@@ -27,15 +27,23 @@ pipeline {
         stage("Vagrant Up") {
             steps {
                 // TODO: Vagrant up --noprovision, install hoot from daily develop RPMs
-                sh "vagrant up ${params.Box} --provider aws"
+                sh "vagrant up ${params.Box} --provider aws --provision-with hoot"
```

Comment:
Would we even need to provision this box at all?  The RPM should install all dependencies on a bare centos 7 box.

Reply:
@brianhatchl The UI tests require at least chrome and that doesn't come from the hoot RPM so there is some level of provisioning still required even after the hoot RPM install (I believe).  I'll try a few permutations locally and see if I can come up with the minimum provisioning required.

Reply:
Ok, true.  My concern is that the `hoot` provision does quite a lot, and is probably duplicative of what the rpm installs (pg, tomcat, etc).
Maybe we need a hoot-ui-test rpm package?